### DISCUSSION TITLE
Odd difference between template and my forked repo

### DISCUSSION THREAD

**Discussion by torreymortenson** [alshedivat/al-folio#239] [Q&A] [answered]
Alright so I'm relatively new to jekyll but was able to get the repo forked and the site running on GH pages. However, what I'm seeing is that the format of the page is different between my forked instance and the base instance. Please see attached images. I don't think I changed any files when I forked but I could have messed something up. Thanks! my site is currently running on torreymortenson.github.io

![mypage](https://user-images.githubusercontent.com/36110135/113968842-23e42980-97f1-11eb-9090-b82172e32e12.JPG)
![basetemplate](https://user-images.githubusercontent.com/36110135/113968843-247cc000-97f1-11eb-9463-1b20e2081233.JPG)

**Comment by alshedivat** [comment, marked as answer]
Based on screenshots, it looks like CSS/JS is not loading correctly. This happens when `url` and `baseurl` are not set correctly in `_config.yml`, which results in incorrect relative paths to CSS/JS. For user websites deployed to GitHub Pages, both `url` and `baseurl` fields must be left empty.
